cpl.gemspec

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require_relative "lib/cpl/version"
+
+Gem::Specification.new do |spec|
+  spec.name    = "cpl"
+  spec.version = Cpl::VERSION
+  spec.authors = ["Justin Gordon", "Sergey Tarasov"]
+  spec.email   = ["justin@shakacode.com", "dzirtusss@gmail.com"]
+
+  spec.summary     = "Heroku to Control Plane"
+  spec.description = "Helper CLI for migrating from Heroku to Control Plane"
+  spec.homepage    = "https://github.com/shakacode/heroku-to-control-plane"
+  spec.license     = "MIT"
+
+  spec.required_ruby_version = ">= 2.7.0"
+
```

Comment:
Don't include specs or anything else unnecessary.

Markdown files? do other gems include them?

Reply:
https://github.com/shakacode/react_on_rails/blob/master/react_on_rails.gemspec#L19-L21

Reply:
Done in a800686427262da3ba11fe7aa820991c4b60ab3e.